crates/workspace/src/workspace.rs

```diff
@@ -6620,38 +6641,105 @@ pub fn client_side_decorations(element: impl IntoElement, cx: &mut WindowContext
                 .size_full()
                 .child(element),
         )
+        .map(|div| match decorations {
```

Comment:
Was this code duplicated accidentally? :thinking:

Reply:
Nope, that's the bit that sets the cursor style. We had to remove it from the code above

Reply:
Hmm, that makes sense. But looks like the old code above is still there:
1st: https://github.com/zed-industries/zed/blob/0395b02a881bc920e74dba916e4028653b9cb58d/crates/workspace/src/workspace.rs#L6500-L6543
2nd: https://github.com/zed-industries/zed/blob/0395b02a881bc920e74dba916e4028653b9cb58d/crates/workspace/src/workspace.rs#L6644-L6685

Reply:
Ooh, I see now, thank you!